service/controller/chart/resource/release/update.go

```diff
@@ -243,10 +244,15 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 	}
 
 	if isReleaseModified(currentReleaseState, desiredReleaseState) {
-		r.logger.Debugf(ctx, "the %#q release has to be updated", desiredReleaseState.Name)
+		opt := cmp.FilterPath(func(p cmp.Path) bool {
+			return p.String() == "Values"
+		}, cmp.Ignore())
+
+		if diff := cmp.Diff(currentReleaseState, desiredReleaseState, opt); diff != "" {
+			fmt.Printf("release %#q have to be updated, (-current +desired):\n%s", cr.Name, diff)
+		}
+
 		return &desiredReleaseState, nil
-	} else {
-		r.logger.Debugf(ctx, "the %#q release does not have to be updated", desiredReleaseState.Name)
```

Comment:
I don't think it has any value to print messages like above. I need to see the whole sentence to just see that it's not gonna do anything.

Reply:
Good point it doesn't add much value. 👍